src/robot-cli/command-listing.h

```diff
@@ -30,12 +32,14 @@
 *******************************************************************************/
 
 /*******************************************************************************
-*                               Variables
+*                               Variables 
 *******************************************************************************/
 
 static const command_t commandArr[] = {
     COMMAND_COMMANDS
     DC_MOTOR_COMMANDS
+    DC_MOTOR_ONE_COMMANDS
```

Comment:
More of a comment than a point of review as it has been done correctly in this case, but we should try to keep the `commandArr` entries in alphabetical order as more cli modules are built up.

Reply:
Agreed